evalml/tests/conftest.py

```diff
@@ -589,14 +591,21 @@ def clone(self):
 
 
 @pytest.fixture
-def logistic_regression_binary_pipeline_class():
+def logistic_regression_binary_component_graph():
+    return [
```

Comment:
I think it's fine to create a new fixture, but if we're using it in conjunction with logistic_regression_binary_pipeline_class, we could either remove this and call logistic_regression_binary_pipeline_class.component_graph or have this be defined as `logistic_regression_binary_pipeline_class.component_graph`.